stdlib/public/Distributed/DistributedActorSystem.swift

```diff
@@ -154,21 +147,20 @@ extension DistributedActorSystem {
   /// is that thanks to this approach it can avoid any existential boxing, and can serve the most
   /// latency sensitive-use-cases.
   public func executeDistributedTarget<Act, ResultHandler>(
-      on actor: Act,
-      mangledTargetName: String,
-      invocation: Self.Invocation,
-      handler: ResultHandler
+    on actor: Act,
+    mangledTargetName: String,
+    invocation: inout Invocation,
```

Comment:
Does invocation have to be `inout` here? I'd think that we don't really want to mutate it at all after it has been constructed...

Reply:
Related to the `mutating` discussion we had -- will clean that up tho then this will take the InvocationDecoder and it'd be `inout` i think so we can do the mutating decodeNext on it :)

Reply:
Sounds good!